embedded-nal-async/rust-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+# Before upgrading check that everything is available on all tier1 targets here:
+# https://rust-lang.github.io/rustup-components-history
+[toolchain]
+channel = "nightly-2022-04-24"
+components = [ "rust-src", "rustfmt" ]
+targets = [ "thumbv7em-none-eabi", "thumbv7m-none-eabi", "thumbv6m-none-eabi", "thumbv7em-none-eabihf", "thumbv8m.main-none-eabihf", "wasm32-unknown-unknown" ]
```

Comment:
How is this file supposed to be used? May be worth documenting somewhere as it looks somewhat non-standard.

Reply:
I can remove this file, it was useful to check that the traits compiled, but CI uses a different mechanism anyway so removing it is fine I think.